app/services/dropbox/pull_service.rb

```diff
@@ -0,0 +1,100 @@
+module Dropbox
+  class PullService < Operation
+    def execute
+      changes = calculate_changes
+
+      create_or_update(changes[:create] + changes[:update])
+      delete(changes[:delete])
+    end
+
+    private
+
+    def create_or_update(to_create)
+      to_create.each do |entry|
+        path, entry_hsh = entry
```

Comment:
Useless assignment to variable - `path`. Use `_` or `_path` as a variable name to indicate that it won't be used.